node/Makefile

```diff
@@ -527,19 +528,20 @@ $(WINDOWS_ARCHIVE_ROOT)/libs/hns/License.txt:
 	wget -P $(WINDOWS_ARCHIVE_ROOT)/libs/hns/ $(MICROSOFT_SDN_GITHUB_RAW_URL)/License.txt
 
 ## Download NSSM.
-windows-packaging/nssm-$(WINDOWS_NSSM_VERSION).zip:
-	wget -O windows-packaging/nssm-$(WINDOWS_NSSM_VERSION).zip https://nssm.cc/ci/nssm-$(WINDOWS_NSSM_VERSION).zip
+windows-packaging/nssm.zip:
+	wget -O windows-packaging/nssm.zip https://nssm.cc/ci/nssm-$(WINDOWS_NSSM_VERSION).zip
 
-build-windows-archive: $(WINDOWS_ARCHIVE_FILES) windows-packaging/nssm-$(WINDOWS_NSSM_VERSION).zip
+build-windows-archive: $(WINDOWS_ARCHIVE_FILES) windows-packaging/nssm.zip
 	# To be as atomic as possible, we re-do work like unpacking NSSM here.
 	-rm -f "$(WINDOWS_ARCHIVE)"
-	-rm -rf $(WINDOWS_ARCHIVE_ROOT)/nssm-$(WINDOWS_NSSM_VERSION)
+	-rm -rf $(WINDOWS_ARCHIVE_ROOT)/nssm
 	mkdir -p dist
 	cd windows-packaging && \
 	sha256sum --check nssm.sha256sum && \
 	cd CalicoWindows && \
-	unzip  ../nssm-$(WINDOWS_NSSM_VERSION).zip \
+	unzip  ../nssm.zip \
 	       -x 'nssm-$(WINDOWS_NSSM_VERSION)/src/*' && \
+	mv nssm-$(WINDOWS_NSSM_VERSION) nssm && \
```

Comment:
This will make sure that regardless of the nssm versions, it will always be inflated to CalicoWindows\nssm directory